vllm/lora/models.py

```diff
@@ -26,9 +26,12 @@
                              parse_fine_tuned_lora_name, replace_submodule)
 from vllm.model_executor.models.interfaces import SupportsLoRA
 from vllm.model_executor.models.utils import PPMissingLayer
-from vllm.utils import is_pin_memory_available
+from vllm.utils import is_pin_memory_available, get_device
```

Comment:
Where are we using get_device? If not needed, please delete.

Reply:
get_device; is used in models.py; it is already part of habana_main; however this import was not part of the original rebase; hence putting it back; else we will get runtime error.